src/main/java/com/mlaf/hu/models/Measurement.java

```diff
@@ -13,6 +13,13 @@
     private double value;
     private CircularFifoQueue<Double> readings = new CircularFifoQueue<>(100);
```

Comment:
Hoezo max 100?

Reply:
Geen doorgronde reden. We zouden dit kunnen omzetten naar een config property. @aacoba  @rogierpijpers  : mee eens?

Reply:
Prima